Files/BaseLayout.cs

```diff
@@ -253,7 +265,16 @@ private void JumpTimer_Tick(object sender, object e)
 
         protected abstract void InitializeCommandsViewModel();
 
-        protected abstract IEnumerable GetAllItems();
+        protected IEnumerable<ListedItem> GetAllItems()
+        {
+            if (CollectionViewSource.IsSourceGrouped)
+            {
+                // add all items from each group to the new list
+                return (CollectionViewSource.Source as BulkConcurrentObservableCollection<GroupedCollection<ListedItem>>)?.SelectMany(g => g);
```

Comment:
@winston-de are you ok with this change?

Reply:
Looks much cleaner, fine by me.